src/components/AggregationPanel/AggregationPanel.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import LoaderDots from 'components/LoaderDots/LoaderDots.react';
+import {
+  TextElement,
+  KeyValueElement,
+  TableElement,
+  ImageElement,
+  VideoElement,
+  AudioElement,
+  ButtonElement,
+} from './AggregationPanelComponents';
+import styles from './AggregationPanel.scss';
+
+const AggregationPanel = ({
+  data,
+  isLoadingCloudFunction,
+  showAggregatedData,
+  setErrorAggregatedData,
+  errorAggregatedData,
+  showNote,
+  setSelectedObjectId,
+  selectedObjectId
+}) => {
+  return (
+    <>
+      {selectedObjectId && isLoadingCloudFunction && showAggregatedData ? (
+        <div className={styles.center}>
+          <LoaderDots />
+        </div>
+      ) : selectedObjectId && showAggregatedData && Object.keys(data).length !== 0 && Object.keys(errorAggregatedData).length === 0 ? (
```

Comment:
Minor: storing this long condition check into a variable at the top might make the code easier to read. It might be also a good case for `useMemo`. Same on the line `26`.